### PR TITLE
README: reorder the branch description

### PR DESCRIPTION
**Pull request by praiskup**
It seems more useful to describe the most useful branches first.

### CODE REVIEW

**Review by Conan-Kudo — APPROVED** [review]

File: mock/README.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ Mock source tarballs: https://github.com/rpm-software-management/mock/releases
...
 
- * `mock-1.0` - This branch was used for EL-5 and there will be no changes.
- * `mock-1.3` - This branch is in security-fixes-only mode and is used for EL-6.
- * `mock-1.4` - This branch is in bug-fixes-only mode.
- * `mock-2` - This branch is used for Mock v2.x, for EL-7, bug-fixes-only mode.
  * `main` - This is currently Mock v3.x and is used for releasing and
    development.  If you want to send patches, you probably want this branch.
+ * `mock-2` - This branch is used for Mock v2.x, for EL-7, bug-fixes-only mode.
+ * `mock-1.4` - This branch is in bug-fixes-only mode.
+ * `mock-1.3` - This branch is in security-fixes-only mode and is used for EL-6.
```
Comment (Conan-Kudo) [inline mock/README.md:23]: EL6 is EOL, so we should mark this dead too, right?

Comment (praiskup) [reply mock/README.md:23]: ah, that's right

File: mock/README.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ Mock source tarballs: https://github.com/rpm-software-management/mock/releases
...
 Mock currently has one active branch plus `main`.
 
- * `mock-1.0` - This branch was used for EL-5 and there will be no changes.
- * `mock-1.3` - This branch is in security-fixes-only mode and is used for EL-6.
- * `mock-1.4` - This branch is in bug-fixes-only mode.
- * `mock-2` - This branch is used for Mock v2.x, for EL-7, bug-fixes-only mode.
  * `main` - This is currently Mock v3.x and is used for releasing and
    development.  If you want to send patches, you probably want this branch.
+ * `mock-2` - This branch is used for Mock v2.x, for EL-7, bug-fixes-only mode.
+ * `mock-1.4` - This branch is in bug-fixes-only mode.
```
Comment (Conan-Kudo) [inline mock/README.md:22]: Who is still using this branch?

Comment (praiskup) [reply mock/README.md:22]: Good question, ideally none.  I don't think we'll need an update there these days.  Updating.

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**